# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsDragService.h
# Commit: f78431f1b071
# Full Hash: f78431f1b0712b491f165891d74cc1c6d1cfdc39
# Author: stransky <stransky@redhat.com>
# Date: 2024-12-07 09:10:49
# Regressor Bug: 1913643
# File Overlap Count: 1
# Description:
#   Bug 1913643 [Linux] Drag&Drop check received data r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231376
# ==============================================================================

diff -r 3288c2058dd7 -r f78431f1b071 widget/gtk/nsDragService.h
--- a/widget/gtk/nsDragService.h	Fri Dec 06 23:25:27 2024 +0000
+++ b/widget/gtk/nsDragService.h	Fri Dec 06 23:55:07 2024 +0000
@@ -59,6 +59,8 @@
 
   int GetURIsNum() const;
 
+  bool IsDataValid() const;
+
 #ifdef MOZ_LOGGING
   void Print() const;
 #endif
